Documentation/Extensions/Management.rst

```diff
@@ -105,9 +105,9 @@ Then run `composer require` to the install the local extension:
 
 .. code-block:: shell
 
-   composer require vendor/my-local-extension
+   composer require my-local-vendor/my-local-extension
```

Comment:
This change does not make sense, it's a standard to use vendor for the vendor part see also the Composer docs!

Reply:
Why does this not make sense?
Nobody uses "vendor".

Reply:
Please read the Composer docs, the package name consists of two parts, the vendor name and the project name see https://getcomposer.org/doc/04-schema.md#name

Reply:
Yes, but the vendor name can be any name. An example with "vendor" as name is confusing to me.

Reply:
While I agree that `vendor` is a commonly used vendor name we should also keep in mind that this is a [tutorial](https://documentation.divio.com/tutorials/):

> Tutorials are lessons that take the reader by the hand through a series of steps to complete a project of some kind. They are what your project needs in order to show a beginner that they can achieve something with it.

So we need to assume that people with no knowledge of Composer will go to through this and it _may_ make sense to reduce the slightest hint of confusion. So this could be discussed at least.

(AFAIR `acme/` is also a common example vendor.)

Reply:
Here `vendor-name` is used.

https://composer.json.jolicode.com